dspace-api/src/main/java/org/dspace/sort/OrderFormatText.java

```diff
@@ -21,6 +22,7 @@
 {
 	{
 		filters = new TextFilter[] { new DecomposeDiactritics(),
+			                         new StripDiacritics(),
```

Comment:
Again, not sure if we should have ` DecomposeDiactritics()` and `StripDiacritics()` together.

Reply:
Resolved via discussion above.